backend-models/service-tests/profileservice-tests.md

```diff
@@ -82,8 +99,17 @@ End pre-
 		$password = "Kong";
```

Comment:
One thing i noticed here is that you actually have to provide the wrong password to get a response from profileservice.php, as well as the new password has to fulfill the password requirements (min 8 letters, one lowercase, one uppercase and a number) Which is why providing Kong won't do anything with profileservice.php

Reply:
The output is correct though, i just couldn't get there the info provided here